.github/workflows/ci_linux.yml

```
@@ -106,3 +106,48 @@ jobs:
       artifact_run_id: "${{ inputs.artifact_run_id != '' && inputs.artifact_run_id || github.run_id }}"
       artifact_group: ${{ inputs.artifact_group }}
       package_version: ${{ inputs.rocm_package_version }}
+
+  parse_package_version:
+    name: Parse package version
+    runs-on: ubuntu-latest
```

We never use latest and always pin to a specific version:
```suggestion
    runs-on: ubuntu-24.04
```